includes/abbreviations.md

```diff
@@ -8,26 +8,30 @@
 *[IFC]: interim final rule with comment period
 *[API criteria]: Certification criteria at § 170.315(g)(7) - (10)
 *[SVAP]: Standards Version Advancement Process
+*[USCDI]: United States Core Data for Interoperability
 
 
 <!-- Standards Referenced -->
 *[SMART IG]: SMART Application Launch Framework Implementation Guide Release 1.0.0
 *[FHIR]: HL7® Fast Healthcare Interoperability Resources
 
 *[a)(1]: Health Level 7 (HL7®) Version 4.0.1 Fast Healthcare Interoperability Resources Specification (FHIR®) Release 4, October 30, 2019
-*[a)(2]: FHIR® US Core Implementation Guide STU V3.1.1
-*[a)(3]: HL7® SMART Application Launch Framework Implementation Guide Release 1.0.0
+*[b)(1]: FHIR® US Core Implementation Guide STU V3.1.1
+*[b)(2]: FHIR® US Core Implementation Guide STU V6.1.0
```

Comment:
Revise US Core 3.1.1 reference to § 170.215(b)(1)(i)
Revise US Core 6.1 reference to § 170.215(b)(1)(ii)

Reply:
https://github.com/onc-healthit/api-resource-guide/commit/88bb512dd0764bbcbef8523d6d0d76c266d59c6a